karyon2-admin/src/main/java/netflix/adminresources/AdminResourcesContainer.java

```diff
@@ -113,11 +114,13 @@ public void init() throws Exception {
                 }
 
                 server = new Server(adminContainerConfig.listenPort());
+                final List<Filter> additionaFilters = adminContainerConfig.additionalFilters();
 
                 // redirect filter based on configurable RedirectRules
                 final Context rootHandler = new Context();
                 rootHandler.setContextPath("/");
                 rootHandler.addFilter(new FilterHolder(adminResourceInjector.getInstance(RedirectFilter.class)), "/*", Handler.DEFAULT);
+                applyAdditionalFilters(rootHandler, additionaFilters);
```

Comment:
Would this not break redirects if the filter is an auth filter?  Do we need auth on the redirect rules since no content is actually served.